### PR TITLE
68 fix footer styling amt

### LINKED ISSUE #68: Fix footer styling
- Fix footer styling at intermediate screen sizes

### PR DESCRIPTION
**Pull request by allison-truhlar**
### What changes did you make?
- Changes to the Footer.jsx and Footer.css.

### Why did you make these changes?
- To fix styling at intermediate screen sizes that made the list of designer and developer names display in columns that were difficult to read/didn't look good.
- To make the Footer fixed to the bottom of the viewport

### Screenshots:

<details><summary>Before</summary>

Screenshot 2023-11-15 at 12 08 52 PM

</details> 

<details><summary>After</summary>

Screenshot 2023-11-15 at 12 09 52 PM

</details>

### CODE REVIEW

File: src/components/footer/Footer.jsx [hunk 1]
```diff
@@ -3,46 +3,45 @@ import './Footer.css';
...
               {' '}
-              <span>
-                <a href="https://github.com/Emimint" target="blank">Emimint</a>
-              </span>
+              <a href="https://github.com/Emimint" target="blank">Emimint</a>
               {' '}
-              <span>
-                <a href="https://github.com/komalgill0310" target="blank">Komal</a>
-              </span>
+              <a href="https://github.com/komalgill0310" target="blank">Komal</a>
```
Comment (AOA19) [inline src/components/footer/Footer.jsx]: @allison-truhlar The value for the `target` attribute should be `_blank`. Right now  when you click on each of our names it doesn't open in a new tab.

**Comment by AOA19** [comment]
@allison-truhlar The footer looks great! I just have one small suggestion, instead of our names being underlined by default maybe they can have an underline effect when hovered over? 

Also I was going to change this later but since you're already working it, do you mind changing my name to match my Github username (AOA19)?

**Comment by allison-truhlar** [comment]
@AOA19 Thanks for these catches/suggestions! 

- I added target="blank" and confirmed all the links open in a new tab. 
- I changed "Alexandra" to "AOA19." 
- I made the links underlined on hover (let me know if you were thinking of something different than what it looks like!) 
- Finally, I noticed that I should have made the footer "sticky," not "fixed," so that it is always at the bottom of the recipe container div, and not overlaid on top of it when it gets filled up with recipe cards.

**Comment by AOA19** [comment]
@allison-truhlar 
- The links look good and the hover effect is exactly what I meant. And thank you for changing my name for me!
- As for the position of the footer, I'm not sure what it looks like on your end, but now that the position changed from `fixed` to `sticky` the footer is now floating when the recipe cards aren't visible. 

- Footer with no search 
footer-nosearch

- Footer with search
footer-search

**Comment by allison-truhlar** [comment]
@AOA19 Good catch - I'll work on it!

**Comment by allison-truhlar** [comment]
@AOA19 I think the footer should stick to the bottom of the page correctly now - it looks okay on my end. Could you please double-check? Thanks!

**Review by AOA19 — APPROVED** [review]
@allison-truhlar The footer is sticking to the bottom now. I'll go ahead and merge these changes.

**Pull request merged**